### DISCUSSION TITLE
script-tags.liquid escaping '@' to '%40'

### DISCUSSION THREAD

**Discussion by qedric** [hayes0724/shopify-packer#79] [Q&A]
Hi there,

the script tag snippets are being generated with '@' symbol escaped - 

After building, I need to go into the dist folder and edit script-tags.liquid from this:

`<script type="text/javascript" src="{{ 'vendor%40layout.theme.js' | asset_url }}" defer="defer"></script>`

to this:

`<script type="text/javascript" src="{{ 'vendor@layout.theme.js' | asset_url }}" defer="defer"></script>`

Is this related to shopify-packer and something I can prevent ?

**Comment by mrpacman101** [comment]
I have noticed that this indeed is an issue for the CSS files too. 
Following...

**Comment by hayes0724** [comment]
@mrpacman101 @qedric 

I had applied a previous fix for js files and forgot to include css to. This is fixed in v2.1.6 and commit 73480959
